src/components/GlobeOptimized.jsx

```diff
@@ -334,7 +334,10 @@ export default function GlobeOptimized(props) {
 
   function dragged(event) {
     const { dx, dy } = event;
-
+    let newParams = new URLSearchParams(window.location.search);
```

Comment:
We need `debounce` here to limit the # of times we change browser's history. Otherwise the application will not be usable if we change the url per drag event.